R/methods.R

```diff
@@ -61,6 +75,16 @@ summary.epinowcast <- function(object, type = c(
 #' @inheritParams enw_plot_nowcast_quantiles
 #' @return `ggplot2` object
 #' @export
+#' @examples
+#' nowcast <- enw_example("nowcast")
+#' latest_obs <- enw_example("obs")
+#'
+#' # Plot nowcast
+#' plot(nowcast, latest_obs = latest_obs, type = "nowcast")
+#'
+#' # Plot posterior predictions by reference date
+#' plot(nowcast, type = "posterior_prediction") +
+#'  ggplot2::facet_wrap(ggplot2::vars(reference_date), scales = "free")
 plot.epinowcast <- function(x, latest_obs = NULL, type = c(
```

Comment:
arguably, `ggplot.epinowcast` is more appropriate here.

Reply:
also, to emphasize previously offered opinion: the plotting stuff should be a separate package.

Reply:
Yes I agree. This depends on developer time and I think is only a good idea when moving out of the experimental stage. If you go multi-repo to early you can tie yourself into a dev nightmare. 

The plan is to do this with all pre and post processing at some point (just not yet)

Reply:
> arguably, ggplot.epinowcast is more appropriate here.

Agree, but its very standard practice to use `ggplot2` inside `plot` calls.